src/geometry/SphericalSegmentGroup.ts

```diff
@@ -137,14 +75,14 @@ export default class SphericalSegmentGroup extends PrimitiveGroup {
     box.makeEmpty();
     normalMatrix.setFromMatrix4(matrix);
     transformedNormal
-      .copy(this.getNormal(globalNormal, index))
+      .copy(this.data.getVector3('normal', globalNormal, index))
       .applyMatrix3(normalMatrix)
       .normalize();
     const scaling = matrix.getMaxScaleOnAxis();
-    const radius = scaling * this.getRadius(index);
-    const height = scaling * this.getHeight(index);
+    const radius = scaling * this.data.getNumber('radiusA', index);
+    const height = scaling * this.data.getNumber('heightA', index);
 
-    sphereCenter.copy(this.getCenter(globalCenter, index)).applyMatrix4(matrix);
+    sphereCenter.copy(this.data.getVector3('centerA', globalCenter, index)).applyMatrix4(matrix);
```

Comment:
We should call it just `center` here too, no?

Reply:
Fixed